share/sysbuild/image_configurations/MAIN_image_default.cmake

```diff
@@ -51,4 +51,13 @@ if(SB_CONFIG_BOOTLOADER_MCUBOOT)
       set_config_bool(${ZCMAKE_APPLICATION} CONFIG_RETENTION_BOOT_MODE y)
     endif()
   endif()
+
+  if(SB_CONFIG_BOOT_ENCRYPTION)
+    if(SB_CONFIG_BOOT_ENCRYPTION_ALG_AES_128)
+      set_config_bool(${ZCMAKE_APPLICATION} CONFIG_MCUBOOT_ENCRYPTION_ALG_AES_128 y)
+    elseif(SB_CONFIG_BOOT_ENCRYPTION_ALG_AES_256)
+      set_config_bool(${ZCMAKE_APPLICATION} CONFIG_MCUBOOT_ENCRYPTION_ALG_AES_256 y)
```

Comment:
Please consider adding the new `forced` options inside the sdk-nrf docs:
https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/app_dev/config_and_build/sysbuild/sysbuild_forced_options.rst